apps/frontend/src/pages/FormPage.tsx

```diff
@@ -84,13 +96,15 @@ const FormPage: React.FC = () => {
 
             <Form>
               {!showBranchMedical && !showBranchState && (
-                <FormPageSections
-                  sectionNum={step}
-                  isMedicalProfessional={isMedicalProfessional}
-                  setIsMedicalProfessional={setIsMedicalProfessional}
-                  isValidState={isValidState}
-                  setIsValidState={setIsValidState}
-                />
+                <FormSection title={sectionTitles[step]}>
```

Comment:
Can you wrap the two <Message> components below in this same FormSection, the title can just be an empty string: ""